exercises/shared/.docs/help.md

```diff
@@ -0,0 +1,15 @@
+# Help
+
+# Getting started
+
+Please refer to the [installation](https://exercism.io/tracks/purescript/installation)
+and [learning](https://exercism.io/tracks/purescript/learning) help pages.
```

Comment:
With v3, we'll have nice docs pages (the links don't work now but will work later).

```suggestion
Please refer to the [installation](https://exercism.io/docs/tracks/purescript/installation)
and [learning](https://exercism.io/docs/tracks//purescript/learning) help pages.
```

Reply:
> (the links don't work now but will work later).

I'm for leaving as-is until "later" and creating an issue to update when they start working. Let me know if that works.

Reply:
Yeah that's fine. They're being redirected anyway.

Reply:
Created https://github.com/exercism/purescript/issues/214